java/src/jmri/jmrit/logixng/tools/swing/EditCommentDialog.java

```diff
@@ -65,17 +65,15 @@ public String showDialog(String comment) {
         buttonCancel.addActionListener((ActionEvent e) -> {
             cancelPressed(null);
         });
-//        cancel.setToolTipText(Bundle.getMessage("CancelLogixButtonHint"));      // NOI18N
-        buttonCancel.setToolTipText("CancelLogixButtonHint");      // NOI18N
+        buttonCancel.setToolTipText(Bundle.getMessage("CancelLogixButtonHint"));      // NOI18N
```

Comment:
Thank you for doing this!

But I'm pretty sure this change will not work. `CancelLogixButtonHint` is defined in `jmri/jmrit/beantable/BeanTableBundle.properties` but the LogixNG classes are in `jmri/jmrit/logixng` so they have not access to that property.

Also, `CancelLogixButtonHint` should be `CancelLogixNGButtonHint`.

This comment also applies to other similar changes in this PR.

Reply:
Thanks, indeed it was. I changed all occurrences of the Cancel button tooltip to LogixNG_CancelButtonHint and added the key in java/src/jmri/jmrit/logixng/LogixNGBundle.properties
Let's wait for the tests to pass